CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 The following is a curated list of changes in the Enact moonstone module, newest changes on the top.
 
+## [unreleased]
+
+### Fixed
+
+- `moonstone/Viewport` to not pass `controlsRef` and `controlsMeasurements` to ui/ViewManager
```

Comment:
I think we don't have to write a changelog for this. :)

Reply:
👍